cluster/saltbase/salt/kubelet/default

```diff
@@ -84,4 +84,4 @@
   {% set pod_cidr = "--pod-cidr=" + grains['cbr-cidr'] %}
 {% endif %} 
 
-DAEMON_ARGS="{{daemon_args}} {{api_servers_with_port}} {{debugging_handlers}} {{hostname_override}} {{cloud_provider}} {{config}} --allow_privileged={{pillar['allow_privileged']}} {{pillar['log_level']}} {{cluster_dns}} {{cluster_domain}} {{docker_root}} {{kubelet_root}} {{configure_cbr0}} {{cgroup_root}} {{system_container}} {{pod_cidr}}"
+DAEMON_ARGS="{{daemon_args}} {{api_servers_with_port}} {{debugging_handlers}} {{hostname_override}} {{cloud_provider}} {{config}} --allow-privileged=true {{pillar['log_level']}} {{cluster_dns}} {{cluster_domain}} {{docker_root}} {{kubelet_root}} {{configure_cbr0}} {{cgroup_root}} {{system_container}} {{pod_cidr}}"
```

Comment:
Set it here: cluster/saltbase/pillar/privilege.sls

And also:

contrib/init/systemd/environ/config

contrib/ansible/roles/kubernetes/templates/config.j2

cluster/gce/coreos/node.yaml

cluster/aws/coreos/node.yaml

docs/high-availability/kube-apiserver.manifest

docs/getting-started-guides/centos/centos_manual_config.md

docs/getting-started-guides/coreos/cloud-configs/node.yaml


and maybe more